.github/workflows/continuous-integration.yml

```
@@ -47,7 +47,7 @@ jobs:
       - name: Install Dependencies
         run: yarn --immutable
       - name: Build test dependencies
-        run: yarn turbo run build --filter=@sapphire/async-queue --filter=@sapphire/*utilities --filter=@sapphire/duration --filter=@sapphire/fetch --filter=@sapphire/lexure --filter=@sapphire/ratelimits --filter=@sapphire/result --filter=@sapphire/timer-manager  --filter=@sapphire/stopwatch --no-deps
+        run: yarn turbo run build --filter=@sapphire/async-queue --filter=@sapphire/bitfield --filter=@sapphire/*utilities --filter=@sapphire/duration --filter=@sapphire/fetch --filter=@sapphire/lexure --filter=@sapphire/ratelimits --filter=@sapphire/result --filter=@sapphire/timer-manager  --filter=@sapphire/stopwatch --no-deps
```

```suggestion
        run: yarn turbo run build --filter="@sapphire/async-queue" --filter="@sapphire/*utilities" --filter="@sapphire/duration" --filter="@sapphire/fetch" --filter="@sapphire/lexure" --filter="@sapphire/ratelimits" --filter="@sapphire/result" --filter="@sapphire/timer-manager"  --filter="@sapphire/stopwatch" --no-deps
```

bitfield only has to be specified here if other packages, or their dependencies, directly depend on bitfield which would then cause type checking errors when tests are ran. Seeing as that is not the case, we don't need to specify it here. 

Also wrapped the packages in quotes to finally get rid of the annoyance where this command cannot be simply copy pasted to a terminal to test because of `*utilities` <_<